docs/preset-env.md

```diff
@@ -302,7 +328,38 @@ var b = new Map();
 
 #### `useBuiltIns: false`
 
-Don't add polyfills automatically per file, or transform `import "@babel/polyfill"` to individual polyfills.
+Don't add polyfills automatically per file, or transform `import "core-js"` to individual polyfills.
```

Comment:
Again, I think we should be careful about wholesale removing `@babel/polyfill` references as we don't want to give the impression that support is completely gone (breaking change).